v2/googlecloud-to-googlecloud/pom.xml

```diff
@@ -19,6 +19,23 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <repositories>
+    <repository>
+      <id>artifact-registry</id>
+      <url>${env.BEAM_ARTIFACT_REGISTRY_URI}</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+    </repository>
+    <repository>
+      <!-- Required for io.confluent:kafka-avro-serializer artifact -->
+      <id>confluent</id>
+      <url>https://packages.confluent.io/maven/</url>
+    </repository>
+  </repositories>
```

Comment:
Can we add this through a profile? In some build contexts, it might fail due to no internet access (PM me for more details).